.chloggen/header-setter-transport-security.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Not sure I would classify this as a breaking change. Perhaps "bug_fix" is more appropriate?